nix/sources.json

```diff
@@ -34,5 +34,17 @@
         "type": "tarball",
         "url": "https://github.com/rhinofi/yarn-berry-cjs/archive/75a5282d9606586d7c2f9e38b07f352cdeab46e3.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "rhino-core": {
+        "branch": "dev",
+        "description": "Instructions to deploy the required services for DeversiFi Exchange",
```

Comment:
"DeversiFi Exchange"

Reply:
good catch, but this is taken from https://github.com/rhinofi/rhino-core, so would have to be updated there first